pkg/miniogw/integration_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package miniogw
+
+import (
+	"context"
+	"flag"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/minio/cli"
+	minio "github.com/minio/minio/cmd"
+	"github.com/stretchr/testify/assert"
+
+	"storj.io/storj/internal/identity"
+	"storj.io/storj/internal/s3client"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/pkg/cfgstruct"
+	"storj.io/storj/pkg/miniogw/logging"
+	"storj.io/storj/pkg/provider"
+)
+
+func TestUploadDownload(t *testing.T) {
+	var (
+		apiKey     = "apiKey"
+		encKey     = "encKey"
```

Comment:
Since you are using both `apiKey` and `encKey` only once, you can put the values directly there.